packages/pigeon/lib/swift_generator.dart

```diff
@@ -43,6 +43,44 @@ class SwiftOptions {
 /// Calculates the name of the codec that will be generated for [api].
 String _getCodecName(Api api) => '${api.name}Codec';
 
+/// Returns a new [Method] using the swift function signature from [func],
+/// ie the name of function e the strings between the semicolons.
```

Comment:
Part of this sentence is missing (I'm not sure what the "e" was intended to say).

Reply:
Fixed